package.json

```diff
@@ -13,7 +13,9 @@
     "node": "^10.14.1"
   },
   "scripts": {
-    "start": "node build-system/pr-check.js"
+    "start": "node build-system/pr-check.js",
+    "lint": "eslint .",
+    "fix": "eslint . --fix"
```

Comment:
This change SGTM assuming we're good with `npm` scripts in this repo. (`amphtml` uses `gulp`, primarily because of a heavy reliance on its file streaming functionality.)

/cc @ampproject/wg-infra

Reply:
I'm ok with sticking with npm scripts.

Reply:
Cool, resolving.